.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
+dist: trusty
+
+go:
+  - 1.7
+
+install:
+  - go get github.com/Sirupsen/logrus
```

Comment:
I vendored the dependencies first, but removed the commit  again as it blew up my PR. How do you do that at DO, do you prefer vendoring or rather install from head?